chainercv/utils/image_utils.py

```diff
@@ -2,22 +2,23 @@
 from PIL import Image
 
 
-def read_image_as_array(path, dtype=np.uint8, copy=True):
+def read_image_as_array(path, dtype=np.uint8, copy=True, force_color=False):
```

Comment:
Since images seem to follow data type protocol used in transforms, shouldn't the default value of `dtype` be `np.float32`?